internal/cli/atlas/clusters/add.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clusters
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type AddOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	name  string
+	store store.SampleDataAdder
+}
+
+func (opts *AddOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+var addTmpl = "Loading sample dataset into {{.ClusterName}}.\n"
```

Comment:
[nit] you should return the id so users can query the status later

Reply:
The `id` is not available 
`{
  "clusterName": "Cluster0",
  "createDate": "2021-04-06T11:44:12Z",
  "id": "",
  "state": "WORKING"
}`

Reply:
we also are not going to implement the get status as it is not part of the scope doc.

Reply:
interesting you mention is not there, which lead me to leave this comment https://github.com/mongodb/go-client-mongodb-atlas/pull/195#discussion_r607871186

Reply:
> we also are not going to implement the get status as it is not part of the scope doc.

Maybe not today users will be thankful if we don't change the output later when we do implement it

Reply:
updated